reference/import_export/connectors.rst

```diff
@@ -0,0 +1,21 @@
+Akeneo Native Connector
+=================
```

Comment:
formatting issue here? you should have same number of "=" than the title contains characters